src/less/InfoBarWidget.less

```diff
@@ -0,0 +1,145 @@
+@import 'mixins';
+
+@spinner-size: 12px;
+
+.ext-wwt-infoBarWidget {
+	position:-webkit-sticky;
+	position: sticky;
```

Comment:
It may be worth noting this isn't supported in IE. I for one could care less :stuck_out_tongue_closed_eyes: `position: sticky` looks so buttery smooth compared to JS implementations, so I think we should keep it.

I don't think we need the `-webkit-sticky` though, since this isn't needed as of Safari 7.1

Reply:
It didn't work for me in chrome without it. It might have had a separate issue, but I wanted to be a bit more extra careful here. 

I'll test it again, see if it was something else that didn't work.